spec/models/wikipedia_artist_fetcher_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+require 'vcr'
```

Comment:
@solipet you shouldn't need to require VCR because it's a gem and Rails will require all gems by default. Checkout [config/application.rb](https://github.com/BaltimorePublicArtCommons/baltimore_public_art_commons/blob/f2580bc3a90c8547fea4aa7c0ecf45f2b004257f/config/application.rb#L10-L12).